database/scripts/schema.py

```diff
@@ -0,0 +1,46 @@
+import psycopg2, time, os
```

Comment:
Using python psycopg2 client is unnecessary. 
Just use psql: 
``` sh
psql -d "postgresql://$DATABASE_USER:$DATABASE_PASSWORD@$DATABASE_HOST_NAME:$PORT_NUMBER/$DATABASE_NAME" -f generate_report.psql
```

Reply:
Python or shell it's not a big deal. If we want to make or add changes later.